modules/products/bitbucket/helm_release.tf

```diff
@@ -0,0 +1,67 @@
+# Install Helm chart for Bitbucket Data Center.
+
+resource "helm_release" "bitbucket" {
+  name       = local.product_name
+  namespace  = var.namespace
+  repository = local.helm_chart_repository
+  chart      = local.product_name
+  version    = local.bitbucket_helm_chart_version
+
+  values = [
+    yamlencode({
+      replicaCount = 1,
+      bitbucket = {
+        clustering = {
+          enabled = true
+        }
+        resources = {
+          jvm = {
+            maxHeap = local.bitbucket_software_resources.maxHeap
+            minHeap = local.bitbucket_software_resources.minHeap
+          }
+          container = {
+            requests = {
+              cpu    = local.bitbucket_software_resources.cpu
+              memory = local.bitbucket_software_resources.mem
+            }
+          }
+        }
+        sysadminCredentials = {
+          secretName = kubernetes_secret.admin_secret.metadata[0].name
```

Comment:
It doesn't need to be part of this PR but Bitbucket has the option to set up the DisplayName as well, it could be added as a variable with auto-generated default:

`bitbucket.displayName`

https://github.com/atlassian/data-center-helm-charts/blob/main/src/main/charts/bitbucket/values.yaml#L567

Reply:
When you say `auto-generated default`, do you mean something like `Bitbucket(<environment_name>)`?

Reply:
Yes, something like that. But you don't need to worry in this PR, maybe just adding a subtask to the user story.

Reply:
Created [TFKUBE-330](https://bulldog.internal.atlassian.com/browse/TFKUBE-330).